src/main/java/com/karankumar/bookproject/backend/entity/Book.java

```diff
@@ -63,6 +63,12 @@
     )
     @JoinColumn(name = "author_id", referencedColumnName = "ID")
     private Author author;
+    
+    @ManyToOne(cascade =
```

Comment:
Could you please make this a many-to-many relationship (a book can have multiple publishers if, for example, it is in the public domain)?

Reply:
I appreciate I said one-to-many before, but having looked into this, I think many-to-many is better